meta-mel/conf/distro/mel.conf

```diff
@@ -112,8 +112,8 @@ do_image_wic[recrdeptask] += "do_populate_lic"
 # on do_populate_lic
 do_image_complete[recrdeptask] += "do_populate_lic"
 
-# Quadruple the normal. 'du' is not a good way to really see how much
-# space will be needed and fails badly as the fs size grows.
+# Quadruple the normal extra space in an image. 'du' is not a good way to really
+# see how much space will be needed and fails badly as the fs size grows.
 IMAGE_ROOTFS_EXTRA_SPACE = "40960"
```

Comment:
Now that we do wic images we've an ability add extra space to the partition from the wic definition. We should probably think over doing this from a single point or else we'll be adding additional space multiple times.

Reply:
Hmm, perhaps. Will wic automatically resize the filesystem image it's pointed to to match the partition size, or does that rely on the first boot partition resizing?

Reply:
wic does that automatically if we use the --extra-space or --overhead option. Ref: https://docs.yoctoproject.org/ref-manual/kickstart.html#command-part-or-partition and does not depend on the first boot resizing.

Reply:
Opened an issue in the repo as a reminder